specification/network/resource-manager/Microsoft.Network/stable/2019-07-01/applicationGateway.json

```diff
@@ -1657,6 +1657,11 @@
             "modelAsString": true
           }
         },
+        "priority": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Priority of the request routing rule."
```

Comment:
LGTM, but you could also add `minimum` and `maximum` parameters (I see in backend model that priority could be between 1 and 20000)

Reply:
Added with exclusive as false